### DISCUSSION TITLE
Make local storage the default

### DISCUSSION THREAD

**Discussion by olfek** [Authenticator-Extension/Authenticator#1111] [Ideas]
### Describe the issue

Some people, like myself for example, may not want to sync data with the browser account.

### Browser

Firefox

### Browser Version

117.0.1 (64-bit)

### Extension Version

6.3.5

**Comment by mymindstorm** [comment]
Switch storage to local in settings and it will not sync.

**Comment by olfek** [comment]
@mymindstorm is it local by default?

**Comment by olfek** [comment]
just checked, doesn't look like it, I think browser sync should be off by default

**Reply by frostilm** [reply]
I don't like this idea. Reasons: 
1. It's quite inconvenient. 
2. Doesn't really make a difference since they are already encrypted.
